cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -199,7 +199,7 @@ def render_and_launch_ingress(kube_dns):
         hookenv.open_port(443)
     else:
         hookenv.log('Deleting the http backend and ingress.')
-        kubectl_manifest('delete', '/etc/kubernetes/addons/default-http-backend.yaml')
+        kubectl_manifest('delete', '/etc/kubernetes/addons/default-http-backend.yaml')  # noqa
```

Comment:
if you split this on the comma, and put it under `'delete'` it will properly be within the 80 col span and not require a fudge `# noqa`